src/AbstractServer.php

```diff
@@ -13,7 +13,7 @@
 /**
  * Abstract Server implementation
  */
-abstract class AbstractServer implements Server
+abstract class AbstractServer implements ServerInterface
```

Comment:
This is BC break, we cannot change the interface here. We need to keep the previous interface usage, and change it in v3.

Reply:
Undoing BC break by reverting implementation of new interface class.